src/utils/utils.js

```diff
@@ -1,11 +1,19 @@
 import axios from 'axios';
 
-const getArticles = () => {
-  return axios
-    .get('https://nc-news-kx4n.onrender.com/api/articles')
-    .then(({ data }) => {
-      return data.articles;
-    });
+const getArticles = query => {
+  if (query) {
+    return axios
+      .get(`https://nc-news-kx4n.onrender.com/api/articles?topic=${query}`)
+      .then(({ data }) => {
+        return data.articles;
+      });
+  } else {
+    return axios
+      .get('https://nc-news-kx4n.onrender.com/api/articles')
+      .then(({ data }) => {
+        return data.articles;
+      });
```

Comment:
This is correct and good to use the same request function as getting all the articles but you don't need the conditional logic for two separate get requests. Instead [look into axios params](https://notes.northcoders.com/courses/js-front-end/axios#axios-params) to see how to take care of this in a cleaner way.